errors/handler.go

```diff
@@ -57,28 +59,52 @@ func LogErrorInRawSpan(rawSpan *tracer.RawSpan, err **errors.Error) {
 }
 
 // Gets the current stack frames array
-func GetCurrentStackFrames(skip int) []StackFrames {
+func getCurrentStackFrames(skip int) []StackFrames {
+	sourceRoot := instrumentation.GetSourceRoot()
 	skip = skip + 1
 	err := errors.New(nil)
 	errStack := err.StackFrames()
 	nLength := len(errStack) - skip
 	if nLength < 0 {
 		return nil
 	}
-	stackFrames := make([]StackFrames, nLength)
+	stackFrames := make([]StackFrames, 0)
 	for idx, frame := range errStack {
 		if idx >= skip {
-			stackFrames[idx-skip] = StackFrames{
+			if len(stackFrames) == 0 {
```

Comment:
Good one. 
Do we want to implement this in all agents? @undefinedlabs/agents

Reply:
I did that to skip internal stack frames from the http and db instrumentation, but don't know if this is the best approach... @fermayo ?

Reply:
this algorithm was reverted.